cmd/registry/config-example.yml

```diff
@@ -11,6 +11,10 @@ http:
   addr: :5000
   headers:
     X-Content-Type-Options: [nosniff]
+auth:
+  htpasswd:
+    realm: basic-realm
+    path: /auth/htpasswd
```

Comment:
would prefer to use `/var/lib/registry` or something under `/etc` for the examples